src/com/google/javascript/jscomp/js/es6/symbol.js

```diff
@@ -46,8 +46,30 @@ $jscomp.symbolCounter_ = 0;
  * @suppress {reportUnknownTypes}
  */
 $jscomp.Symbol = function(opt_description) {
-  return /** @type {symbol} */ (
-      $jscomp.SYMBOL_PREFIX + (opt_description || '') + ($jscomp.symbolCounter_++));
+  // List of ECMAScript 2015 "Well-known Symbols"
+  // See Table 1 in https://www.ecma-international.org/ecma-262/6.0/#sec-well-known-symbols
+  /** @const @dict @type {!Object<string, string>} */
+  var WELL_KNOWN_SYMBOLS = {
+    'hasInstance': '@@hasInstance',
```

Comment:
If we include this here, it implies that we actually support the hasInstance symbol. But if we don't also update usages of the `instanceof` operator then we aren't really supporting it, we're just making it so code that checks for the existence of that symbol will incorrectly think we are.